_data/toc.yaml

```diff
@@ -22,43 +22,49 @@ guides:
   section:
   - path: /engine/installation/
     title: Install Docker
-  - sectiontitle: Docker Edge
+  - sectiontitle: Docker EE
```

Comment:
I think for this spot you should spell out "Enterprise Edition" and "Community Edition" since they are the first instances of the terms. It's okay to use the acronyms downlevel though. :)